apps/common/index.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  name: 'common',
+  steps: {
+    '/start': {
+      template: 'start',
+      next: 'ima/continue-form'
+    },
```

Comment:
I don't think we need the start route in the common app as it comes from the IMA app

Reply:
Start route removed